pinot-segment-local/src/main/java/org/apache/pinot/segment/local/realtime/impl/invertedindex/RealtimeLuceneTextIndex.java

```diff
@@ -205,6 +210,24 @@ public void close() {
     }
   }
 
+  /**
+   * Returns the partition from a segment name, or zero if partition is not a valid integer
```

Comment:
Can we add this method to the existing SegmentNameUtils class?

Reply:
done